app/src/main/java/org/mozilla/focus/MainActivity.kt

```diff
@@ -115,6 +119,21 @@ class MainActivity : LocaleAwareAppCompatActivity(), BrowserFragmentCallbacks, U
             }
         }
 
+        val bannerLayout: View? = findViewById(R.id.bannerLayout)
+        if (bannerLayout != null) {
+            val bannerMoreInfoButton: Button? = bannerLayout.findViewById(R.id.bannerMoreInfoButton)
+            if (bannerMoreInfoButton != null) {
+                bannerMoreInfoButton.setOnClickListener {
+                    ScreenController.showBrowserScreenForUrl(supportFragmentManager, SupportUtils.getSumoURLForTopic(this, "firefox-echo-show"), Source.NONE)
+                }
+            }
+
+            val bannerTextView: TextView? = bannerLayout.findViewById(R.id.bannerTextView)
+            if (bannerTextView != null) {
+                bannerTextView.text = getString(R.string.banner_text)
```

Comment:
This used to include the `Html.fromHtml()` call but since these messages won't be styled I removed that.